README.md

```diff
@@ -1,4 +1,4 @@
-# Adobe Analytics Android Extension
+# Adobe Experience Platform - Analytics extension for Android
```

Comment:
we can keep`Extension` instead of `extension` to match all our headers in other files